custom_components/ha_washdata/learning.py

```
@@ -848,6 +865,38 @@ async def async_resolve_pending_from_label(
+            if cycle is not None:
+                rv = cycle.get("ml_review") if isinstance(cycle.get("ml_review"), dict) else {}
+                already_reviewed = bool(rv.get("reviewed_at"))
+                needs_review = (
+                    rv.get("label") in ("uncertain", "review")
+                    or rv.get("quality") in ("uncertain", "review")
+                    or cycle.get("status") in ("force_stopped", "interrupted")
+                )
+                if needs_review and not already_reviewed:
+                    try:
+                        await self.profile_store.set_cycle_review(cycle_id)
+                    except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)